src/common/JSHandle.ts

```diff
@@ -849,7 +846,7 @@ export class ElementHandle<
     await elementHandle.dispose();
 
     /**
-     * This as is a little unfortunate but helps TS understand the behavour of
+     * This `as` is a little unfortunate but helps TS understand the behaviour of
```

Comment:
While we're at it:

```suggestion
     * This `as` is a little unfortunate but helps TS understand the behavior of
```

The source code is supposed to be `en-US`, not `en-GB` (sorry @jackfranklin!)